language/bytecode-verifier/src/control_flow_graph.rs

```diff
@@ -102,6 +102,7 @@ impl VMControlFlowGraph {
     }
 
     fn is_end_of_block(pc: CodeOffset, code: &[Bytecode], block_ids: &Set<BlockId>) -> bool {
+        // XXX: Overflow of PC is possible here!
```

Comment:
Did we resolve this issue based on the discussion before the holidays?  The tentative conclusion was that the bytecode generator is not modeling the serializer.

Reply:
We did indeed resolve this, I just haven't gotten around to update yet, but I'll be removing this comment. 

One thing worth thinking about, but that we don't need to address now, is whether or not the bytecode verifier should assume these type of preconditions (e.g. that the code passed to it it also passed the deserializer).

Reply:
Good point.  The "bytecode verifier" is organized as a sequence of passes.  These passes are partially-ordered in that a pass depends on all passes to successfully complete before being invoked.  This organization is likely inevitable.  The bytecode deserializer could be either thought of as the first pass or we could implement a "pass 0" in the bytecode verifier that does the checks performed by the deserializer.  

@dariorussi : What is your opinion?